protos/feast/core/FeatureSet.proto

```diff
@@ -60,6 +60,9 @@ message FeatureSetSpec {
     // Optional. Source on which feature rows can be found.
     // If not set, source will be set to the default value configured in Feast Core.
     Source source = 6;
+
+    // User defined metadata
+    map<string,string> labels = 8;
```

Comment:
Should it be 7?

Reply:
7 is already taken:
`string project = 7;`

Reply:
Ah my bad, missed that it got added all the way at the top of the message definition.